lib/active_model_serializers/serializable_resource.rb

```diff
@@ -2,7 +2,9 @@
 
 module ActiveModelSerializers
   class SerializableResource
-    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links, :serialization_context, :key_transform])
+    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta,
+                                   :meta_key, :links, :root,
+                                   :serialization_context, :key_transform])
```

Comment:
As opposed to https://github.com/rails-api/active_model_serializers/pull/1782/files#r66382756 👍 here since it's the adapter's job to build the JSON response. the serializer just provides 'attributes' and 'associations' and hints as to what it's root should be, when needed